vrecord

```diff
@@ -2257,9 +2286,11 @@ while getopts ":hc:erpaxvI:O:i:G" opt ; do
         p) RUNTYPE="passthrough" ;;
         a) RUNTYPE="audiopassthrough" ;;
         x) RUNTYPE="reset" ;;
+        n) RUNTYPE="resetenvironment" ;;
```

Comment:
worth adding to `-h` output?

Reply:
yah - thanks for catching that! I'll add